lib/feature/one_hot_encoder.cc

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+#include <fstream>
+#include <string>
+
+#include "clink/feature/one_hot_encoder.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+using namespace clink;
+using namespace std;
+
+OneHotEncoderModel::OneHotEncoderModel(){}
```

Comment:
Since this is an empty method, would it be simpler to define it in the header file directly?

And as a good practice, could we mark constructor without parameters with the `explicit` keyword?

Reply:
OK. I'll make the change.